content/docs/traffic-management/destination-types/backends/lambda/creds-secret.md

```diff
@@ -145,19 +152,18 @@ At this point, {{< reuse "docs/snippets/product-name.md" >}} is routing directly
 1. Delete the `lambda` HTTPRoute and `lambda` Backend.
    
    ```sh
-   kubectl delete HTTPRoute lambda -n {{< reuse "docs/snippets/ns-system.md" >}}
-   kubectl delete Backend lambda -n {{< reuse "docs/snippets/ns-system.md" >}}
+   kubectl delete HTTPRoute lambda -n {{% reuse "docs/snippets/ns-system.md" %}}
+   kubectl delete Backend lambda -n {{% reuse "docs/snippets/ns-system.md" %}}
    ```
 
 2. Delete the `aws-creds` secret.
    
    ```sh
-   kubectl delete secret aws-creds -n {{< reuse "docs/snippets/ns-system.md" >}}
+   kubectl delete secret aws-creds -n {{% reuse "docs/snippets/ns-system.md" %}}
    ```
 
 3. Use the AWS Lambda console to delete the `echo` test function.
 
 ## Known limitations
 
-- Currently, the only parameter that is supported for the `backendRefs.filters.extensionRef` field in the HTTPRoute resource is the name of the Lambda function. Additional parameters, such as `wrapAsApiGateway`, `unwrapAsApiGateway`, or `invocationStyle`, are not supported.
-- Authenticating with your AWS account by using IAM Roles for Service Accounts (IRSA) is currently unsupported.
+- Currently, the only parameter that is supported for the `backendRefs.filters.extensionRef` field in the HTTPRoute resource is the name of the Lambda function. Additional parameters, such as `wrapAsApiGateway`, `unwrapAsApiGateway`, or `invocationStyle`, are not supported.
```

Comment:
Since there is only one, maybe don't use bullet points?

Reply:
Also, how would people even know there are other parameters? are these upstream K8s GW API parameters in the HTTProute?